recipes/expat/all/test_v1_package/conanfile.py

```diff
@@ -1,18 +1,18 @@
-# pylint: skip-file
-from conans import ConanFile, CMake, tools
+from conans import ConanFile, CMake
+from conan.tools.build import cross_building
 import os
 
 
-class TestPackageConan(ConanFile):
+class TestPackageV1Conan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
-    generators = "cmake", "cmake_find_package"
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
ok but please add tests to Find module please (see freetype recipe)

Reply:
@SpaceIm 
Thank you for your comment.
I'm going to add it.